README.md

```diff
@@ -22,5 +22,64 @@ For more information, check out the WAX [blog](https://wax.io/blog/how-the-wax-r
     npm install
     npm run test
     ```
+
+### Register bandwidth payer
+
```

Comment:
Let's add a similar summary line here as the error log section below:
*WAX RNG allows dapps to pay for their own bandwidth, which can prevent your dapp from losing service during times of high activity on the rng contract. In the future, WAX will reduce the free bandwidth available for apps, so it is a good idea to migrate to this to ensure your dapp is always up with respect to random number generation*